customer-wallet/src/containers/Customer/CustomerWallet/CustomerWalletForm/index.js

```diff
@@ -47,12 +48,15 @@ class CustomerWalletFormWrapper extends Component {
   handleChangeAddress = (activeAddress) => this.setState({ activeAddress })
 
   handleSubmit = ({ toAddress, amountBtc }) => {
-    this.setState({ isDisabledSubmit: true }, () => {
-      const { rating, convertBitsToSatoshi } = this.props
+    this.setState({ isDisabledSubmit: true }, async () => {
+      const { rating, convertBTCToSatoshi } = this.props
```

Comment:
@fazo96 I think there is an error here, it should be convertBitsToSatoshi.
@Venumteam - can you confirm?

Reply:
@kulpreet you are right, there should be `convertBitsToSatoshi` which we already get from input form

Reply:
Sorry guys, I made a mistake when rebasing

Reply:
@fazo96 that's ok. Can you push a fix for this and then we can merge this.